docs/run-platform-server/configuring.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: Configuring
+sidebar_position: 30
+---
+
+import { Alert } from "@site/src/components/Alert";
+import { CodeExample } from "@site/src/components/CodeExample";
+
+Pre-requisites:
+
+- [Installation](./installing.mdx) is completed on host machine, it should have two executables, `stellar-horizon` and `stellar-core`. Per the installation instructions, `stellar-core` should be included in your host o/s PATH as these configurations will result in Horizon searching for `stellar-core` to execute.
+- [Prepare the Database if first time](#prepare-the-database-first-time)
+
+You are now ready to setup configuration parameters to perform three important, distinct roles:
+
+- **serving requests** a regular web-based HTTP API,
+- **ingesting ledgers** from core nodes of a Stellar network to keep its world-view up to date, and
+- **transaction submission** for sending transactions to the Stellar network.
+
+You choose from one of two following deployment modes to perform the roles and proceed with configuration parameters to that means. A note on configuration parameters, the same parameter can be provided either as command line argument or an environment variable.
+
+## Single Instance Deployment
+
+You run `stellar-horizon` in a single o/s process and it will perform all three roles at same time.
```

Comment:
```suggestion
In this simpler deployment variant, you run `stellar-horizon` in a single OS process and it will perform all three roles at same time.
```

Reply:
what's your opinion on using second person pronouns such as 'you'? briefly looking at tech/academic guides, they suggest rephrasing to 1st/3rd, the former looks widely used in docs, and assuming best to stay consistent for overall style?

Reply:
I'm not going to rock the boat on this, and follow existing precedence of referring to second person.